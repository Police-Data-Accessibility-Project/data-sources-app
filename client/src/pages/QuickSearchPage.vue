<template>
	<main class="p-4 md:p-0 md:my-24">
		<QuickSearchForm />
	</main>
</template>

<script>
import { QuickSearchForm } from "pdap-design-system";

export default {
	name: "QuickSearchPage",
	components: {
		QuickSearchForm,
	},
};
</script>
