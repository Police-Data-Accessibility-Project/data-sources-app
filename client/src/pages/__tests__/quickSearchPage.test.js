import { shallowMount } from '@vue/test-utils';
import QuickSearchPage from '../QuickSearchPage.vue';
import { describe, expect, test } from 'vitest';

describe('QuickSearchPage', () => {
	test('is a Vue instance', () => {
		const wrapper = shallowMount(QuickSearchPage);
		expect(wrapper.vm).toBeTruthy();
		expect(wrapper.html()).toMatchSnapshot();
	});
});

// Shallow mounting this test for now. It's 1. not really testing much, and 2. erroring on `getRoutes`
// TODO: write an actual test for this.
