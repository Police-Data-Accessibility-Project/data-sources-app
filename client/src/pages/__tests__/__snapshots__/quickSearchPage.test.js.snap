// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`QuickSearchPage > is a Vue instance 1`] = `
<main class="p-4 md:p-0 md:my-24">
  <quick-search-form-stub baseurlforredirect="https://data-sources.pdap.io"></quick-search-form-stub>
</main>
`;
